classes/classes/Scenes/Combat/Combat.as

```diff
@@ -1790,13 +1806,9 @@ package classes.Scenes.Combat
 			hpDisplay = Math.floor(monster.HP) + " / " + monster.eMaxHP() + " (" + (int(math * 1000) / 10) + "%)";
 			lustDisplay = Math.floor(monster.lust) + " / " + monster.eMaxLust();;
 
-			//trace("trying to show monster image!");
-			if (monster.imageName != "")
-			{
-				var monsterName:String = "monster-" + monster.imageName;
-				//trace("Monster name = ", monsterName);
-				outputText(images.showImage(monsterName), false,false);
-			}
+			//imageText set in beginCombat()
+			outputText(imageText, false,false);
```

Comment:
Could you remove the `, false,false` from that call, please? Its the default and just yet another relic from Fenfolk's spaghetti code.